src/containers/Home/Home.js

```diff
@@ -41,19 +68,22 @@ export default class Home extends Component {
       <div>
         <Helmet title="The Ultimate Web API Framework"/>
 
-        <div id="incredible">
+        <div id="incredible" className="incredible" style={{minHeight: this.state.windowHeight}}>
           <section className="container">
-            <img src={logoImage} alt="API Platform" width={300}/>
-            <p>The ultimate <a href="http://php.net" target="_blank">PHP</a> framework to build modern web APIs.<br /></p>
+            <h1><img src={logoImage} alt="API Platform" height={400}/></h1>
+            <p>The ultimate <a href="http://php.net" target="_blank">PHP</a> framework to build modern web APIs.</p>
 
             <Link className="btn btn-primary btn-lg" to="/docs/distribution/" role="button"><span className="fa fa-plus-circle"/> Get started</Link>
-            <a className="btn btn-primary btn-lg" href="https://demo.api-platform.com" role="button"><span className="fa fa-mouse-pointer" />Demo</a>
+            <Link className="btn btn-primary btn-lg" to="https://demo.api-platform.com" role="button"><span className="fa fa-mouse-pointer"/>Demo</Link>
+            <a className="btn btn-primary btn-lg" href="https://api.github.com/repos/api-platform/api-platform/zipball" role="button"  target="_blank"><span className="fa fa-download" /> Download</a>
           </section>
         </div>
 
-        <div id="features" className="slide">
+        <HeaderNav inline={true}/>
+
+        <div id="features" className="slide features">
           <section className="container">
-            <h1>Creating Linked Data <a href="https://en.wikipedia.org/wiki/Representational_state_transfer" target="_blank">REST</a> APIs has never been so easy</h1>
+            <h2>Creating Linked Data <a href="https://en.wikipedia.org/wiki/Representational_state_transfer" target="_blank">REST</a> APIs has never been so easy</h2>
```

Comment:
There's a huge debate about that: having one h1 or multiple h1.
In term of SEO and accessibility, having a clear hierarchy is way better. The main info of your website is the title "API Platform" .
In term of accessibility, they can navigate inside of a page through a special menu built by heading level (and I don't think there's a reset on section for it)
The actual code doesn't built hierarchy at all. All infos are on the same level and I think we have to change this